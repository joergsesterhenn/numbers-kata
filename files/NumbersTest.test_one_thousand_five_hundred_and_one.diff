--- NumbersTest.test_one_thousand_five_hundred_and_one.approved.txt
+++ NumbersTest.test_one_thousand_five_hundred_and_one.received.txt
@@ -1 +1 @@
-one thousand, five hundred and one
+onethousand, five hundred and one
