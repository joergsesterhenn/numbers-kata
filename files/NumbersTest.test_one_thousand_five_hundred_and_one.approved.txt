one thousand five hundred and one